src/services/AsyncStorage.ts

```diff
@@ -56,12 +60,43 @@ export const removeCustomDiscipline = async (customDiscipline: string): Promise<
   await setCustomDisciplines(disciplines)
 }
 
+const getFavorites = async (): Promise<Document[]> => {
```

Comment:
The new endpoint for getting words by id is already available, als for live cms, so can be adjusted or if you do not want to do it in this PR, please open a ticket.

Reply:
Querying by ids is not possible yet though isn't it? I think this would be nice, otherwise we need to query all words and then filter in the app. There is already https://issues.tuerantuer.org/browse/LUN-307